test/v1/controllers/AmendReliefInvestmentsControllerSpec.scala

```diff
@@ -50,6 +50,7 @@ class AmendReliefInvestmentsControllerSpec
       lookupService = mockMtdIdLookupService,
       parser = mockAmendReliefInvestmentsRequestParser,
       service = mockService,
+      auditService = mockAuditService,
```

Comment:
You need to add audit mocking in each of the test scenarios like [here](https://github.com/hmrc/self-assessment-bsas-api/blob/5d7b9cb25643a333c14beb8d81b52f2afb59d007/test/v1/controllers/SubmitUkPropertyBsasControllerSpec.scala#L128)